Dockerfile

```diff
@@ -26,4 +26,7 @@ RUN apk --no-cache add ca-certificates libdrm
 RUN apk --no-cache add hwloc --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community
 WORKDIR /root/
 COPY --from=0 /go/bin/k8s-device-plugin .
+
 CMD ["./k8s-device-plugin", "-logtostderr=true", "-stderrthreshold=INFO", "-v=5"]
+ENTRYPOINT ["/k8s-device-plugin"]
```

Comment:
Doesn't this create "/k8s-device-plugin ./k8s-device-plugin -logtostderr=true -stderrthreshold=INFO -v=5"